src/AppBundle/Admin/User/AdministratorAdmin.php

```diff
@@ -42,4 +44,39 @@ protected function configureListFields(ListMapper $list)
             ->add('lastName', null, ['label' => 'form.label_last_name'])
         ;
     }
+
+    /**
+     * @inheritDoc
+     */
+    protected function configureShowFields(ShowMapper $show)
+    {
+        $show
+            ->add('email', null, ['label' => 'form.label_email'])
+            ->add('firstName', null, ['label' => 'form.label_first_name'])
+            ->add('lastName', null, ['label' => 'form.label_last_name'])
+        ;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    protected function configureFormFields(FormMapper $form)
+    {
+        $form
+            ->add('email', null, ['label' => 'form.label_email'])
```

Comment:
Can your add : ->with('fieldset.label_general', ['class' => 'col-md-7']) for email and names